fixtures/log.py

```diff
@@ -44,7 +44,9 @@ def pytest_runtest_logreport(report):
             try:
                 logger().info(
                     "Managed providers: {}".format(
-                        ", ".join(pytest.store.current_appliance.managed_providers)))
+                        ", ".join([
+                            prov.key for prov in pytest.store.current_appliance.managed_providers]))
```

Comment:
Elaborate? The only way I can see this work is having a ProvidersContainer that would be thrown around whenever we work with multiple providers giving you the ability to do `providers.keys` but... we don't want to go that way, it's better to use list comprehension here and there rather than that. Or am I missing something? @psav